content/en/tracing/profiling/java.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Profiling Java Applications
+kind: documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+The Datadog Profiler requires [Java Flight Recorder][1]. The Datadog Profiling library supports the Java JRE 11 and higher of both Oracle JDK and OpenJDK. Early-access versions of Java aren't supported. All JVM-based languages, i.e. Scala, Groovy, Kotlin, Clojure, etc. are supported.
```

Comment:
What's an early-access version of java?

Why not mention zulu 8 given that it's the easiest way for java 8 users (ie _most_ java users) to try profiling today?